### DISCUSSION TITLE
Unable to get public sponsor badge

### DISCUSSION THREAD

**Discussion by jackleeio** [community/community#123393] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I sponsored several users but did not get the public sponsor badge

**Comment by sajadh76** [comment]
Hello @jackleeio 

Based on my knowledge, you have to sponsor open-source contributors. Did you do that?

**Reply by jackleeio** [reply]
Yes, I sponsored an organization and a contributor, but didn't get a sponsor badge, does this seem to be a bug?

<img width="368" alt="Screenshot 2024-05-12 at 11 54 34" src="https://github.com/community/community/assets/20477587/d5013720-cfaa-4217-a2c7-f432c372b637">

**Comment by CiceroLino** [comment]
Strange, anyone who has get a sponsor or have sponsoring any profile, automatically get the badge. Did you try turning off the badges and put on again? How knows if works

And maybe this take some time to you recieve the achievement.

**Comment by bin2209** [comment]
Strange, anyone who has get a sponsor or have sponsoring any profile, automatically get the badge. Did you try turning off the badges and put on again. How knows if works

**Comment by kazmatw** [comment]
Try to sponsor at least 2$ ?

**Reply by CiceroLino** [reply]
Wait, now I remember. @jackleeio  To you catch the badge, you must at least pay 5$

**Reply by jackleeio** [reply]
I have already paid. The strange thing is that the badge can be displayed on the github mobile app, but it cannot be displayed on the github web page, and I have deleted the page cache and tried again, but it still cannot be displayed.
<img width="416" alt="Screenshot 2024-05-12 at 14 43 26" src="https://github.com/community/community/assets/20477587/befe395f-4219-4c7d-a985-1a5bcc861600">

**Reply by CiceroLino** [reply, marked as answer]
# So, it was matter of time to github update your profile.

![Captura de tela de 2024-05-13 15-55-08](https://github.com/community/community/assets/53065315/621727ee-0cbc-4e4d-9b79-be4902cde3ee)

@jackleeio  please close this discussion if possible.
